src/coreclr/jit/gentree.cpp

```diff
@@ -25404,8 +25391,620 @@ GenTree* Compiler::gtNewSimdRoundNode(var_types type, GenTree* op1, CorInfoType
     return gtNewSimdHWIntrinsicNode(type, op1, intrinsic, simdBaseJitType, simdSize);
 }
 
+GenTree* Compiler::gtNewSimdShuffleNodeVariable(
```

Comment:
please add comment section like other functions have in this file (so don't, but it's not an excuse - we try to be consistent)